aidistillery/index_tfidf.py

```diff
@@ -70,9 +58,10 @@ def _add_args(parser):
     parser.add_argument("--hash", action='store_true', default=False,
                         help="Use (stateless) hashing vectorizer")
 
+
 if __name__ == "__main__":
```

Comment:
Keeping this here unlike what we did in the other subcommands. 
I argue that we can keep both the ability to run the script directly AND the subcommand. This is more user friendly and sticks with the easy of having 1 script being able to do 1 thing with the script alone. Distill subcommand ties everything together.
Do you agree @lgalke @vinid ?